app/models/flight.rb

```diff
@@ -2,8 +2,10 @@ class Flight < ActiveRecord::Base
   belongs_to :destination, :class_name => "Airport"
   belongs_to :departure, :class_name => "Airport"
 
+  validates_presence_of :destination_id, :departure_id, :departure_date,
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.